vitrivr-engine-server/src/main/kotlin/org/vitrivr/engine/server/Main.kt

```diff
@@ -47,38 +45,39 @@ fun main(args: Array<String>) {
 
 
         /* Registers Open API plugin. */
-        c.plugins.register(
-            OpenApiPlugin(
-                OpenApiPluginConfiguration()
-                    .withDocumentationPath("/swagger-docs")
-                    .withDefinitionConfiguration { _, u ->
-                        u.withOpenApiInfo { t ->
-                            t.title = "vitrivr engine API"
-                            t.version = "1.0.0"
-                            t.description = "API for the vitrivr engine."
-                        }
-                        u.withSecurity(
+        c.registerPlugin(OpenApiPlugin{
+            it.withDocumentationPath("/openapi.json")
+                .withDefinitionConfiguration{ _, def ->
+                    def.withInfo{i ->
+                        i.title = "vitrivr engine API"
+                        i.version = "1.0.0"
```

Comment:
more like 0.1.0

Reply:
Fair enough, this happens when you follow the migration guide too much...